examples/arkode/C_serial/ark_kpr_mri_1_0_0.01.out

```diff
@@ -66,4 +66,4 @@ Multirate nonlinear Kvaerno-Prothero-Robinson test problem:
 Final Solver Statistics:
    Steps: nsts = 501, nstf = 50601
    u error = 5.052e-05, v error = 7.684e-04, total error = 5.445e-04
-   Total RHS evals:  Fs = 1504,  Ff = 151853
+   Total RHS evals:  Fs = 1504,  Ff = 253055
```

Comment:
This problem setup has zero-valued fast RHS function (i.e., `fast_type == 0`), i.e., the full problem is evolved at the slow time scale.

In the previous version, when `fast_type == 0` (as in this `.out` file), it would manually set the fast Butcher table.  I reorganized the example to add MERK and IMEX-MRI-SR methods, and in so doing I cleaned up how the fast/slow integrators were set up.  Because I touched that part of the example file, I don't recall who requested it, but I believe that a reviewer asked that `fast_type == 0` instead use the default explicit Butcher table from ARKStep for the fast time scale.  That change increased the number of fast RHS evaluations.

Reply:
Actually, I just looked back through the reviews and don't see any request to change the fast Butcher table.  I'll push a simple change that will revert this to use the same manually-specified Butcher table as before.  This will likely result in some CI failures due to `.out` changes.

Reply:
OK, either way is fine by me, I just noticed the big increase and wanted to bring attention to it .